pkg/reconciler/apibinding/apibinding_reconcile.go

```diff
@@ -312,11 +331,15 @@ func (r *workspaceAPIExportReferenceReconciler) reconcile(ctx context.Context, a
 						err,
 					)
 					return reconcileStatusStop, nil // don't retry
+				case !apierrors.IsAlreadyExists(err):
+					// TODO(ncdc): Do we want to set a user-facing condition here?
+					// This case and the next have the same effect (retry). But this one could loop, whereas the default
+					// case should proceed on the next reconcile iteration.
```

Comment:
this can only loop for some limited time, no? You do a get above. So only while stale the cache will return NotFound before and we land here.

Reply:
In other words: retry is fine.

Reply:
Old code/comment - deleted locally 😄